json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -210,8 +210,8 @@ public function build_current_site_response() {
 		return $this->render_response_keys( $response_keys );
 	}
 
-	private function has_blog_access( $token_details, $blog_id ) {
-		if ( is_user_member_of_blog( get_current_user_id(), $blog_id ) ) {
+	private function has_blog_access( $token_details, $wpcom_blog_id ) {
+		if ( is_user_member_of_blog( get_current_user_id(), get_current_blog_id() ) ) {
```

Comment:
Looks like there's no need to pass these params in, as `is_user_member_of_blog()` uses the same ones by default [#](https://github.com/WordPress/WordPress/blob/master/wp-includes/user.php#L709-L711) [#](https://github.com/WordPress/WordPress/blob/master/wp-includes/user.php#L728-L730)

Not a blocker tho